app/peerinfo/peerinfo_test.go

```diff
@@ -23,22 +23,28 @@ func TestPeerInfo(t *testing.T) {
 	now := time.Now()
 	const gitCommit = "1234567"
 
+	// when a major release happens, charon will only be compatible with a single version
+	versions := version.Supported()
+	if len(versions) < 2 {
+		versions = append(versions, versions...)
+	}
```

Comment:
If there is a single version supported we... duplicate it?

Reply:
yes, as a corner case in this test, in order to keep the test working when there will be more than one supported

Reply:
Not the prettiest, but fine by me, beauty is in the eye of the beholder 😄